index.html

```diff
@@ -935,8 +937,9 @@ <h3>2006</h3>
 
 			<!-- Additional nav bar on mobile version so no need to scroll to the top. -->
 			<nav id="nav-personal" class="nav hideondesktop">
-				<a href="#me" class="icon fa-home" style="text-decoration:none"><span>Home</span></a>
-				<a href="#work" class="icon fa-folder active" style="text-decoration:none"><span>Work</span></a>
+
+				<a href="#me" class="icon fa-home active" style="text-decoration:none"><span>Home</span></a>

```

Comment:
I think these might not be correct anymore! did you mean to change them all? can you explain your goal? From memory they were showing as "headers" on mobile version for what "page" the person is on/where they're up to

Reply:
nvm it was easy to fix myself! since I want to see this beautiful slideshow on the website because I need to show it to someone :grin: thanks again for the feature!!!

Reply:
Sorry! Yes they are supposed to be active on the correct tabs, I made the mistake thinking they were all supposed to start at "home"" so I gave it all the "active" when the website started, but It makes more sense that they are active for the specific tabs they are supposed to be.